samples/deployment/vmware/deploy.yml

```diff
@@ -1,52 +1,56 @@
-- name: "Deploy and destroy vFTD in the VMware cloud"
+- name: Deploy vFTD in the VMware cloud
   hosts: localhost
   gather_facts: no
   vars_files:
-    - ./vars.yaml
+    - ./vars.yml
   tasks:
-    - name: "Check OVF presense"
+    - name: Check OVF presense
       stat:
         path: '{{ ovf_file_name }}'
       register: ovf_file_info
       delegate_to: localhost
-    - name: "Download OVF"
+
+    - name: Download OVF
       get_url:
         url: "{{ remote_ovf_location }}"
         dest: "./"
       when: 'ovf_file_info.stat is defined and ovf_file_info.stat.exists == False'
-    - name: "Check VMDK presense"
-        stat:
-          path: '{{ remote_vmdk_location | basename }}'
-        register: vmkdk_file_info
-        delegate_to: localhost
-    - name: "Download VMDK"
+
+    - name: Check VMDK presense
+      stat:
+        path: '{{ remote_vmdk_location | basename }}'
+      register: vmkdk_file_info
+      delegate_to: localhost
+
+    - name: Download VMDK
       get_url:
         url: "{{ remote_vmdk_location }}"
         dest: "./"
       when: 'vmkdk_file_info.stat is defined and vmkdk_file_info.stat.exists == False'
-    - name: "Gather facts about datastore"
+
+    - name: Gather facts about datastore
       vmware_datastore_facts:
         hostname: "{{ vcenter_hostname }}"
         username: "{{ vcenter_username }}"
         password: "{{ vcenter_password }}"
         datacenter: "{{ vcenter_datacenter }}"
       delegate_to: localhost
       register: datastore_list
-    - name: "Create vFTD VM"
+
+    - name: Create vFTD VM
       vmware_deploy_ovf:
         hostname: "{{ vcenter_hostname }}"
         username: "{{ vcenter_username }}"
         password: "{{ vcenter_password }}"
         datacenter: "{{ vcenter_datacenter }}"
         resource_pool: "{{ vcenter_resource_pool }}"
-        datastore: "{{ ( datastore_list.datastores | sort(attribute='freeSpace') | last).name }}"
+        datastore: "{{ ( datastore_list.datastores | sort(attribute='freeSpace', reverse=True) | selectattr('datastore_cluster', 'eq', 'GCP-Pods') | first ).name }}"
```

Comment:
Lets move GCP-Pods in to variable which can should be customised

Reply:
Moved to variable in `vars.yml`. Done